src/Components/Web.JS/dist/Release/blazor.server.js

```diff
@@ -1,25 +1 @@
rits(f,s);for(var c=o(a.prototype),u=0;u<c.length;u++){var l=c[u];f.prototype[l]||(f.prototype[l]=a.prototype[l])}function f(e){if(!(this instanceof f))return new f(e);s.call(this,e),a.call(this,e),e&&!1===e.readable&&(this.readable=!1),e&&!1===e.writable&&(this.writable=!1),this.allowHalfOpen=!0,e&&!1===e.allowHalfOpen&&(this.allowHalfOpen=!1),this.once("end",h)}function h(){this.allowHalfOpen||this._writableState.ended||r.nextTick(d,this)}function d(e){e.end()}Object.defineProperty(f.prototype,"writableHighWaterMark",{enumerable:!1,get:function(){return this._writableState.highWaterMark}}),Object.defineProperty(f.prototype,"destroyed",{get:function(){return void 0!==this._readableState&&void 0!==this._writableState&&(this._readableState.destroyed&&this._writableState.destroyed)},set:function(e){void 0!==this._readableState&&void 0!==this._writableState&&(this._readableState.destroyed=e,this._writableState.destroyed=e)}}),f.prototype._destroy=function(e,t){this.push(null),this.end(),r.nextTick(t,e)}},function(e,t,n){"use strict";n.d(t,"a",(function(){return r})),n.d(t,"b",(function(){return o}));const r={init:function(e,t){t._blazorInputFileNextFileId=0,t.addEventListener("click",(function(){t.value=""})),t.addEventListener("change",(function(){t._blazorFilesById={};const n=Array.prototype.map.call(t.files,(function(e){const n={id:++t._blazorInputFileNextFileId,lastModified:new Date(e.lastModified).toISOString(),name:e.name,size:e.size,contentType:e.type,readPromise:void 0,arrayBuffer:void 0};return t._blazorFilesById[n.id]=n,Object.defineProperty(n,"blob",{value:e}),n}));e.invokeMethodAsync("NotifyChange",n)}))},toImageFile:async function(e,t,n,r,i){const s=o(e,t),a=await new Promise((function(e){const t=new Image;t.onload=function(){e(t)},t.src=URL.createObjectURL(s.blob)})),c=await new Promise((function(e){var t;const o=Math.min(1,r/a.width),s=Math.min(1,i/a.height),c=Math.min(o,s),u=document.createElement("canvas");u.width=Math.round(a.width*c),u.height=Math.round(a.height*c),null===(t=u.getContext("2d"))||void 0===t||t.drawImage(a,0,0,u.width,u.height),u.toBlob(e,n)})),u={id:++e._blazorInputFileNextFileId,lastModified:s.lastModified,name:s.name,size:(null==c?void 0:c.size)||0,contentType:n,readPromise:void 0,arrayBuffer:void 0};return e._blazorFilesById[u.id]=u,Object.defineProperty(u,"blob",{value:c}),u},ensureArrayBufferReadyForSharedMemoryInterop:async function(e,t){const n=await i(e,t);o(e,t).arrayBuffer=n},readFileData:async function(e,t,n,r){const o=await i(e,t);return btoa(String.fromCharCode.apply(null,new Uint8Array(o,n,r)))}};function o(e,t){const n=e._blazorFilesById[t];if(!n)throw new Error(`There is no file with ID ${t}. The file list may have changed.`);return n}function i(e,t){const n=o(e,t);return n.readPromise||(n.readPromise=new Promise((function(e,t){const r=new FileReader;r.onload=function(){e(r.result)},r.onerror=function(e){t(e)},r.readAsArrayBuffer(n.blob)}))),n.readPromise}},function(e,t,n){"use strict";(function(e){
```

Comment:
Unfortunately we do have to include this output file for `blazor.server.js` in source control. It's annoying that we have to, and frequently forces us to resolve merge conflicts. But it's needed due to our source build requirements for certain Linux distributions.

Reply:
File is still there; is just a single line as doesn't need the node.js buffer module pollyfill for the browser